components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesBookmarksStorage.kt

```diff
@@ -199,6 +180,16 @@ open class PlacesBookmarksStorage(context: Context) : PlacesStorage(context), Bo
         return places.importBookmarksFromFennec(dbPath)
     }
 
+    /**
+     * Read pinned sites from Fennec's browser.db file.
+     *
+     * @param dbPath Absolute path to Fennec's browser.db file.
+     * @return A list of [BookmarkNode] which represent pinned sites.
+     */
+    fun readPinnedSitesFromFennec(dbPath: String): List<BookmarkNode> {
+        return places.readPinnedSitesFromFennec(dbPath)
```

Comment:
Oh, I thought they get returned when we import. Has this changed?

Reply:
Yup, A-S changed the API some time ago. Bookmark import now returns a json blob with some stats about how the import went (which we then parse into our telemetry), and pinned sites are a separate method.